Assets/Scripts/AudioTrigger.cs

```diff
@@ -4,11 +4,23 @@
 
 public class AudioTrigger : MonoBehaviour
 {
+    public bool isReturnTrigger = false;
+
     public AudioClip triggerClip;
     public bool isMusicTrigger = false;
     public bool destroyOnPlay = true;
 
-    public bool playSubtitles = true;
     public string[] subtitleText;
     public float[] subtitleTime;
+
+    private void Start()
+    {
+        if (isReturnTrigger)
+            transform.parent.gameObject.SetActive(false);
```

Comment:
Maybe add comment on this? It is not clear from the code what the parent object should be. If the parent is audiotriggers or even worse - terrain, this could cause issues.